### PR TITLE
WIP PR for Rich's licensing backports

### PR DESCRIPTION
**Pull request by mdlinville**
This PR previews the licensing backports. 

Raise PRs against branch `20241029-DOC-10624-backports`, and:
- When you create the PR, it will have HTML previews
- When you merge such a PR, it will show up in this PR

To publish the changes, merge this PR into `main`.

### CODE REVIEW

**Comment by rmloveland** [comment]
@shannonbradshaw and @taroface 

for your convenience: 

- [Rendered preview site (Licensing FAQs page)](https://deploy-preview-19143--cockroachdb-docs.netlify.app/docs/v23.1/licensing-faqs)

**Review by shannonbradshaw — APPROVED** [review]
I moved quickly, but did look at every change. Left one comment.

Great work Rich!

**Comment by rmloveland** [comment]
> I moved quickly, but did look at every change. Left one comment.
> 
> Great work Rich!

thanks @shannonbradshaw !!! I have to fix a busted link due to https://github.com/cockroachdb/docs/pull/19098 i think

File: src/current/_includes/releases/v24.2/v24.2.4.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Release Date: October 17, 2024
 <h3 id="v24-2-4-db-console-changes">DB Console changes</h3>
 
 - Streamlined [metric chart]({% link v24.2/ui-overview.md %}#metrics) legends by removing the name of the chart from labels, where it was an identical prefix for all labels on the chart. [#129359][#129359]
-- The [DB Console]({% link v24.2/ui-overview.md %}) now shows a notification if the cluster has no [Enterprise license]({% link v24.2/enterprise-licensing.md %}) set. Refer to [upcoming license changes](https://www.cockroachlabs.com/enterprise-license-update/) for more information. [#130417][#130417]
+- The [DB Console]({% link v24.2/ui-overview.md %}) now shows a notification if the cluster has no Enterprise license set. Refer to [upcoming license changes](https://www.cockroachlabs.com/enterprise-license-update/) for more information. [#130417][#130417]
```
[the remark this answers is not in the document]
Comment (rmloveland) [reply src/current/_includes/releases/v24.2/v24.2.4.md:32]: (whereas this release note is v24.2.4)

**Pull request merged**